scripts/pylib/twister/twisterlib/harness.py

```diff
@@ -507,6 +487,116 @@ def _parse_report_file(self, report):
             self.instance.reason = 'No tests collected'
 
 
+class Robotframework(TwisterLibraryHarness):
+
+    def configure(self, instance: TestInstance):
+        super(Robotframework, self)._configure(instance, "robot", 'robot_report.xml', 'robot.html')
+
+    def robot_run(self, timeout):
+        self._run(timeout)
+
+    def _get_base_command(self):
+        command = [
+            sys.executable,
+            "-m",
+            "twister_harness.robot_framework.robot_wrapper",
```

Comment:
Pytest plugin (and pytest also) is not used here, only a module, that is located in twister_harness dir. So maybe that module should be moved to outside of pytest-twister-harness plugin?